frontend/app/schemas/personal-informaton-service-schemas.server.ts

```diff
@@ -247,7 +247,7 @@ export const personalInformationSchema = z.object({
       streetName: z.string().optional(),
       apartment: z.string().optional(),
       cityName: z.string().optional(),
-      provinceTerritoryStateId: z.string().optional(),
+      id: z.string().optional(),
```

Comment:
I don't think this field should be renamed - `provinceTerritoryStateId` semantically differs from `id` of a mailing address.

(Same with line 240 above for home address)

Reply:
Fixed, my bad.